src/main/java/core/basesyntax/FileWork.java

```
@@ -8,7 +13,24 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+    private static final String FIRST_CHAR = "w";
+
+    public static String[] readFromFile(String fileName) {
+        StringBuilder sb = new StringBuilder();
+        String text = "";
+        try {
+            text = Files.readString(Paths.get(fileName));
+        } catch (IOException e) {
+            e.printStackTrace();
```

don't ignore exception, re-throw it